scss/_tables.scss

```diff
@@ -81,6 +81,11 @@
   tbody tr {
     @include hover {
       background-color: $table-bg-hover;
+
+      > td,
+      > th {
```

Comment:
It seems Bootstrap's team try to avoid child selector, see this comment for exemple : https://github.com/twbs/bootstrap/pull/18400#r48385183